mico-admin/src/app/toolbar/toolbar.component.html

```diff
@@ -3,4 +3,5 @@
     <a mat-button [routerLink]="['/dashboard']">Dashboard</a>
     <a mat-button [routerLink]="['/app-detail/app-list']">Applications</a>
     <a mat-button [routerLink]="['/service-detail/service-list']">Services</a>
+    <a mat-button target="_blank" rel="noopener" href="http://{{openFaaSIp}}"><span>OpenFaaS UI </span><mat-icon>link</mat-icon></a>
```

Comment:
The current implementation leads to the URL `http://http//40.115.25.83:8080`.
If we change it to `href="{{openFaaSIp}}"` it does not work, because the browser don't open a website with a IP address in a new tab.
I tested several variants to fix that but I found no solution.

Reply:
There seems to be a `:` missing after the second `http`.
For this PR i would prefer the correct link even if ot does not open in a new tab. This beahaviour can be fixed later.

Reply:
I agree and suggest to open the OpenFaaS portal in the same tab, because that actually works. We can add an issue that we want to open it in a new tab.